packages/preset-wind/src/rules/background.ts

```diff
@@ -77,7 +86,9 @@ export const backgroundStyles: Rule[] = [
   [/^(?:bg-gradient-)?from-op(?:acity)?-?(.+)$/, ([, opacity]) => ({ '--un-from-opacity': h.bracket.percent(opacity) })],
   [/^(?:bg-gradient-)?via-op(?:acity)?-?(.+)$/, ([, opacity]) => ({ '--un-via-opacity': h.bracket.percent(opacity) })],
   [/^(?:bg-gradient-)?to-op(?:acity)?-?(.+)$/, ([, opacity]) => ({ '--un-to-opacity': h.bracket.percent(opacity) })],
-
+  [/from-([0-9].+)$/, bgGradientPositionResolver('from')],
```

Comment:
```suggestion
  [/^(from|via|to)-([\d\.]+)%$/, bgGradientPositionResolver('from')],
```

Like this? Should we explicitly ask for it to be end with `%`?

Reply:
Of course, I have adjusted